en/pocket_privacy_policy.md

```diff
@@ -42,7 +42,7 @@ We use cookies and analytics tools for the following purposes:
 
 * Necessary: These cookies are necessary for our website to function and to support our business. They cannot be switched off. For example, we use them to manage your login state, and our ad partners use them to display ads and measure impressions and clicks. You can set your browser to block or alert you about these cookies, but this may cause parts of the site to not work properly. These cookies do not store any personally identifiable information and unless Personalized Advertising Cookies are enabled, ads are non-personalized.
 * Analytics: We place these cookies on your device to help us understand how users engage with our website and product, and to remember your account preferences and language settings. For example, we use Google Analytics and other tools to understand which Pocket webpages are popular. We also use pixels in our emails in order to collect usage information.
-* Personalized Advertising: Some Pocket web pages have ads. If you choose, Pocket ad partners will place advertising cookies on your device to personalize the ads you see here and on other websites. If you play a YouTube video on Pocket, YouTube may place cookies on your device which can be used for personalization. Our partner for personalized ads on our syndicated article pages, Freestar, has its own [Privacy Policy](https://freestar.com/privacy-policy/)
+* Personalized Advertising: Some Pocket web pages have ads. If you choose, Pocket ad partners will place advertising cookies on your device to personalize the ads you see here and on other websites. If you play a YouTube video on Pocket, YouTube may place cookies on your device which can be used for personalization. Your use of YouTube and its content is also subject to the YouTube [Privacy Policy](https://policies.google.com/privacy?hl=en). Our partner for personalized ads on our syndicated article pages, Freestar, has its own [Privacy Policy](https://freestar.com/privacy-policy/)
```

Comment:
We should remove reference to `en` in the URL: https://policies.google.com/privacy

Reply:
OK - will peoples' browsers automatically set to the right version based on their browser settings?  (i.e. if I go to  https://policies.google.com/privacy from a French browser, will it automatically correct the URL to send me to the French version?)

Reply:
Yes. I get Italian in a private window, since my browser is in Italian, and I get English in my standard browsing because I'm logged in to an account which has English set as language.

Reply:
Sounds good, thanks.